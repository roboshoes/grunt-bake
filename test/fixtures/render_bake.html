<!DOCTYPE html>
<html>
	<head></head>
	<body>
		<!--(bake includes/include-one.html _render="foo")-->
		<!--(bake includes/include-two.html _render="bar")-->

		<!--(bake-start _render="foo")-->
		Foo
		<!--(bake-end)-->

		<!--(bake-start _render="bar")-->
		Bar
		<!--(bake-end)-->
	</body>
</html>