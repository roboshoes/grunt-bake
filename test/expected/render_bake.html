<!DOCTYPE html>
<html>
	<head></head>
	<body>
		Include Two



		Bar
	</body>
</html>